### PR TITLE
FragmentTestPrettymuchComplete

### PR DESCRIPTION
**Pull request by AivanD**
- organized the 3 fragments for future testing. same logic
- Finished the FragmentTest.java****
- updated gradle

### CODE REVIEW

File: code/app/src/main/java/com/example/emojibrite/NameScreenFragment.java
Comment (AivanD) [inline code/app/src/main/java/com/example/emojibrite/NameScreenFragment.java:1]: organized it for possible future testing. Same logic

File: code/app/src/main/java/com/example/emojibrite/PreviewScreenFragment.java
Comment (AivanD) [inline code/app/src/main/java/com/example/emojibrite/PreviewScreenFragment.java:1]: organized it for possible future testing. Same logic

File: code/app/src/main/java/com/example/emojibrite/UploadImageScreenFragment.java
Comment (AivanD) [inline code/app/src/main/java/com/example/emojibrite/UploadImageScreenFragment.java:1]: organized it for possible future testing. Same logic

**Review by SnehalRay — COMMENTED** [review]
ggs

**Pull request merged**